static/js/blueslip.js

```diff
@@ -362,6 +362,17 @@ function build_arg_list(msg, more_info) {
     return args;
 }
 
+// Format error stacks using the ErrorStackParser
+// external library
+function getErrorStack(ex) {
+    return ErrorStackParser.parse(ex)
```

Comment:
When I tested this the parser code itself threw an exception, so we need to be very defensive here.  (I'm not sure why my error crashed it.)

Reply:
@showell Could you give me any more details? I made a change from es6 to es5 that was causing tests to fail on my local machine and pushed that out after the initial push. Would you mind testing again? What do you mean by crashed it?